app/__init__.py

```diff
@@ -17,18 +17,34 @@ def create_app(test_config=None):
     if test_config is None:
         app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
             "SQLALCHEMY_DATABASE_URI")
+        app.config["SLACK_API_URI"] = os.environ.get(
+            "SLACK_API_URI")
     else:
         app.config["TESTING"] = True
         app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
             "SQLALCHEMY_TEST_DATABASE_URI")
+        app.config["SLACK_API_URI"] = os.environ.get(
+            "SLACK_API_URI")
 
     # Import models here for Alembic setup
     from app.models.task import Task
     from app.models.goal import Goal
+    from .routes import task_bp
+    from .goal_routes import goal_bp
```

Comment:
Since the `routes.py` only holds Task-related routes, we should rename it to `task_routes.py` and make this import more intuitive!